functions.py

```diff
@@ -524,3 +585,18 @@ def xor(x, y):
 xor.__doc__ = _(
     'xor(x, y), logical xor. Returns True if either x is True \
 (and y is False) or y is True (and x is False), else returns False')
+
+
+def significant(x):
+    x = str(x)
+    if '.' in x:
+        integer, fraction = x.split('.')
+        fraction = fraction.lstrip('0')
+        integer = integer.lstrip('0')
+        integer = integer.rstrip('0')
```

Comment:
You should not be doing a rstrip on the integer part in case it's a decimal input

Reply:
I have changed it.